<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="Button_Select">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Select File</string>
    </property>
   </widget>
   <widget class="QListWidget" name="List_Variable">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>201</width>
      <height>391</height>
     </rect>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::MultiSelection</enum>
    </property>
   </widget>
   <widget class="QWidget" name="Show_Plot" native="true">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>30</y>
      <width>521</width>
      <height>491</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="Layout_Plot"/>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
